src/read/elf/relocation.rs

```diff
@@ -102,12 +102,12 @@ impl<'data, Elf: FileHeader> Iterator for ElfRelocationIterator<'data, Elf> {
     fn next(&mut self) -> Option<Self::Item> {
         match self {
             ElfRelocationIterator::Rel(ref mut i, endian) => {
-                i.next().cloned().map(|r| Crel::from_rel(r, *endian))
+                i.next().cloned().map(|r| Crel::from_rel(&r, *endian))
```

Comment:
```suggestion
                i.next().map(|r| Crel::from_rel(r, *endian))
```
Same below, and cargo fmt.